docs/system-requirements.md

```diff
@@ -0,0 +1,44 @@
+# System requirements
+
+## Server
+Additional to normal Nextcloud requirements the following constraints apply to use Nextcloud Talk:
+
+### HTTPS
+ 
+HTTPS is required to be able to use WebRTC (the video call technic of browsers used by Nextcloud Talk calls).
+
+### Database
+* SQLite: must not be used, to grant a decent experience for chats and calls
+* MySQL/Maria DB: Must enable utf8mb4 support** as per documentation at [Enabling MySQL 4-byte support](https://docs.nextcloud.com/server/latest/admin_manual/configuration_database/mysql_4byte_support.html)
```

Comment:
What are the two `**` behind "utf8mb support" for?

Reply:
I reformated it later, before I used `**` to make it bold